pages/demo/css/css-intern.tsx

```diff
@@ -6,6 +6,7 @@ import Header from '~/components/Header';
 import Footer from '~/components/Footer';
```

Comment:
Same as in the previous files.

We should only modify where to replace with `<FormattedText>`.